### DISCUSSION TITLE
Is there a plan to release 4.4.0 to central?

### DISCUSSION THREAD

**Discussion by pzygielo** [spotbugs/spotbugs#1648] [Q&A] [answered]
RN was published for [4.4.0](https://github.com/spotbugs/spotbugs/releases/tag/4.4.0) but I can't see this version at https://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs.

**Comment by KengoTODA** [comment, marked as answer]
thanks for comment, I forgot handling release procedure at Nexus. Now it's deployed to central, please check it.

**Reply by pzygielo** [reply]
It is in central now indeed and works perfectly fine.
Thank you.

**Comment by hazendaz** [comment]
closing as resolved.
